redis_websocket_api/handler.py

```diff
@@ -131,35 +141,41 @@ def channel_is_allowed(self, channel_name):
     @classmethod
     async def create(cls, redis, websocket, read_timeout=None):
         """Create a handler instance setting up tasks and queues."""
-        self = cls(redis, websocket, read_timeout=read_timeout)
-        self.consumer_task = asyncio.ensure_future(self._websocket_reader())
+
+        if read_timeout:
+            logger.warning(
+                "read_timeout is not used anymore because cleanup is trigered "
+                "immidiatly on connection loss"
```

Comment:
```suggestion
                "read_timeout is not used anymore because cleanup is triggered "
                "immediately on connection loss"
```